xml/Microsoft.Build.BuildEngine/BuildTask.xml

```diff
@@ -16,7 +16,7 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary>Represents a [Task](http://msdn.microsoft.com/library/d82e2485-e5f0-4936-a357-745bacccc299) element in a project.</summary>
+    <summary>Represents a <see href="http://msdn.microsoft.com/library/d82e2485-e5f0-4936-a357-745bacccc299">Task</see> element in a project.</summary>
```

Comment:
This is on docs: https://docs.microsoft.com/en-us/visualstudio/msbuild/task-element-msbuild